src/per_test_config.rs

```diff
@@ -416,16 +407,21 @@ impl TestConfig<'_> {
         let output = crate::core::run_command(&mut cmd)?;
 
         let output = self.check_test_result(&cmd, output)?;
+        
+        let silent = TestConfig {
+            config: self.config.clone(),
+            comments: self.comments,
+            aux_dir: self.aux_dir,
+            status: Box::new(SilentStatus {
+                revision: self.status.revision().to_string(),
+                path: self.status.path().to_path_buf(),
+            }),
+        };
 
         for rev in self.comments() {
             for custom in rev.custom.values() {
                 for flag in &custom.content {
-                    if let Some(result) =
-                        flag.post_test_action(self, &mut cmd, &output, build_manager)?
-                    {
-                        runs.push(result);
-                        return Ok(TestOk::Ok);
-                    }
+                    flag.post_test_action(&silent, &mut cmd, &output, build_manager)?;
                 }
             }
```

Comment:
I ended up making `Flag` things silent

Personally I prefer not having the `.fixed` etc files be their own line in the log, but also timing wise the spinners felt redundant as the main test's/revision's spinner and all of its sub tests would be marked as complete at once after `run_test` returned

Reply:
Let's do that in a separate PR, as I'd rather do some experimentation, and not block this PR on said experimentation

Reply:
Should I have it return a Vec in this PR in that case?

Reply:
Hmm good point. Yea a vec seems best